### ISSUE TITLE
Default PostProcess behavior is changed compared with DETR

### ISSUE THREAD

**Issue by YuyaoXiaoCS** [Atten4Vis/ConditionalDETR#8]
Hi Author, 

In your code: 
`
        prob = out_logits.sigmoid()
        topk_values, topk_indexes = torch.topk(prob.view(out_logits.shape[0], -1), 100, dim=1)
        scores = topk_values
        topk_boxes = topk_indexes // out_logits.shape[2]
        labels = topk_indexes % out_logits.shape[2]
`
In DETR:
`
        prob = F.softmax(out_logits, -1)
        scores, labels = prob[..., :-1].max(-1)
`

This behavior is a little bit strange that you select the top 100 scores within 300 object query and its classes. Which means that one object query is allowed to selected twice with different class. It would be really appreciated if you could give any explanation!

Thanks!
